xorg/PKGBUILD.mesa

```diff
@@ -32,7 +32,7 @@ build() {
     --with-dri-driverdir=/usr/lib/xorg/modules/dri \
     --with-dri-drivers= --with-gallium-drivers=freedreno,swrast \
   --with-egl-platforms=x11 --enable-gles2 --enable-gles1 \
-  --enable-gallium-egl --disable-gallium-llvm --enable-xa --disable-dri3 \
+  --enable-gallium-egl --disable-gallium-llvm --disable-xa --disable-dri3 \
```

Comment:
Enabling XA in mesa from freedesktop.org worked fine for me, btw, performance has significantly improved